earth_enterprise/src/fusion/autoingest/sysman/AssetVersionD.cpp

```diff
@@ -171,14 +171,27 @@ AssetVersionImplD::SetState(AssetDefs::State newstate, bool propagate)
     notify(NFY_DEBUG, "SetState: %s %s",
```

Comment:
Would it be better to just update this notify to include the `state` variable rather than the 3 notifys that you are adding?  Something like:
notify(NFY_DEBUF, "Set State to %s from %s for asset %s", ToString(newstate).c_str(),  ToString(state).c_str(), GetRef().c_str());